om4labs/diags/moc/moc.py

```diff
@@ -100,15 +104,29 @@ def generate_basin_masks(basin_code, basin=None):
     return mask
 
 
-def arguments(cliargs=None):
+class DefaultDictParser(argparse.ArgumentParser):
```

Comment:
move to om4common?

Reply:
Hmm ... I thought I did.  Let me check correct and if not I will.